Makefile

```diff
@@ -0,0 +1,86 @@
+.DEFAULT_GOAL := help
+
+PSYSH_SRC = bin src box.json.dist composer.json composer.lock build/stub
+
+.PHONY: help
+help:
+	@fgrep -h "##" $(MAKEFILE_LIST) | fgrep -v fgrep | sed -e 's/\\$$//' | sed -e 's/##//'
+
+
+##
+## Commands
+##---------------------------------------------------------------------------
+
+clean: 	 ## Clean all created artifacts
+.PHONY: clean
+clean:
+	rm -rf build/
+
+
+build: ## Compile the application into the PHAR
+.PHONY: build
+build: build/psysh.phar build/psysh-compat.phar build/psysh-php54.phar build/psysh-php54-compat.phar
```

Comment:
So the PHP 5.4 PHARs are being built thanks to the Composer `platform.php` option which ensures the deps are installed as everything was on 5.4. So the 4 PHARs are generated here instead of 2 PHARs in two separate builds.

Note that the non PHP 5.4 PHAR is currently built in 7.1, you may want to lower that

Reply:
I think I tried building on 7.2, 7.1 and 7.0, and the one that worked most consistently across PHP versions was 7.0. Let's keep that, for now, and possibly bump it later.